addons/l10n_in/models/res_partner.py

```diff
@@ -28,6 +28,30 @@ class ResPartner(models.Model):
     )
 
     display_pan_warning = fields.Boolean(string="Display pan warning", compute="_compute_display_pan_warning")
+    l10n_in_gst_state_warning = fields.Char(compute="_compute_l10n_in_gst_state_warning")
+
+    @api.depends_context('company')
```

Comment:
If there is no partner set on the partner, we look at all the companies in the company selector
```suggestion
    @api.depends_context('allowed_company_ids')
```

But can't we just add a field dependency on `fiscal_country_codes` instead? (which should in turn propagate the dependency on the context)